php-classes/Slate/UI/Adapters/Courses.php

```diff
@@ -68,11 +68,11 @@ public static function getIcon(\ActiveRecord $Record)
 
     public static function getLinks($context = null)
     {
-        if (empty($_SESSION['User'])) {
+        if (!$_SESSION['User']) {
```

Comment:
@nbey this will still throw a warning if the User key isn't defined, `empty` was the key part of preventing that